README.md

```diff
@@ -155,6 +188,14 @@ Returns an array of results - one item for each item in `patches`. The type of e
 * `remove`, `replace` and `move` - original object that has been removed
 * `add` (only when adding to an array) - index at which item has been inserted (useful when using `-` alias)
 
+#### jsonpatch.applyPatch (`obj` Object, `patch` Patch, `validate` Boolean) : boolean
+
+Available in *json-patch.js* and *json-patch-duplex.js*
+
+Applies a single `patch` on `obj`.
+
+Returns the modified object.
+
```

Comment:
I'd rename it to `applyOperationObject`/`applyOperation` to be more consistent with naming in spec. Then I'd remove argument and variable inside to `operation`